coderd/users_test.go

```diff
@@ -1902,6 +1902,106 @@ func TestGetUsers(t *testing.T) {
 		require.Len(t, res.Users, 1)
 		require.Equal(t, res.Users[0].ID, first.UserID)
 	})
+
+	t.Run("LoginTypeNoneFilter", func(t *testing.T) {
+		t.Parallel()
+		client := coderdtest.New(t, nil)
+		first := coderdtest.CreateFirstUser(t, client)
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
+		defer cancel()
```

Comment:
```suggestion
		ctx := testutil.Context(t, testutil.WaitLong)
```
I see none of the other tests in this file use this util, but we might as well use it here.